### PR TITLE
Implement management connection close

### PR DESCRIPTION
**Pull request by k-wall**
### Type of change

- Enhancement / new feature

### Description

Implement management connection close

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [x] Update CHANGELOG.md

### CODE REVIEW

**Comment by k-wall** [comment]
@lulf @vbusch There will be a separate PR integrating this into the UI.  I'll add a systemtest to that PR giving us end to end test coverage.

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=SmokeTest

**Comment by k-wall** [comment]
@enmasse-ci run tests profile=isolated

File: pkg/consolegraphql/resolvers/resolver_connection.go [hunk 1]
```diff
@@ -163,6 +163,80 @@ func (r *queryResolver) Connections(ctx context.Context, first *int, offset *int
...
+
+		commandDelegate, e := collector.CommandDelegate(token, requestState.ImpersonatedUser)
+		if e != nil {
+			return nil, e
+		}
+
+		e = commandDelegate.CloseConnection(con.ObjectMeta)
+		if e != nil {
+			graphql.AddErrorf(ctx, "connection: '%s' in namespace '%s' could not be closed : %+v", con.Name, con.Namespace, e)
+		}
```
Comment (lulf) [inline pkg/consolegraphql/resolvers/resolver_connection.go:238]: Does commandDelegate.Shutdown() need to be called?

Comment (k-wall) [reply pkg/consolegraphql/resolvers/resolver_connection.go:238]: No, that's not required.  They get cached pkg/consolegraphql/agent/agent.go:53 and tracked by a last use time. `expireCommandDelegates` pkg/consolegraphql/agent/agent.go:307   takes care of shutting down any that sit idle for too long.

**Pull request merged**